pkgs/top-level/python-packages.nix

```diff
@@ -4849,6 +4849,14 @@ in {
       openbabel = (callPackage ../development/libraries/openbabel { python = self.python; });
   };
 
+  smhi = callPackage ../development/python-modules/smhi { };
+
+  homeconnect = callPackage ../development/python-modules/homeconnect { };
+
+  pysmartapp = callPackage ../development/python-modules/pysmartapp { };
+
+  pysmartthings = callPackage ../development/python-modules/pysmartthings { };
+
```

Comment:
Please keep the file alphabetically ordered by adding new entry in their appropriate location.